### PR TITLE
chore(api): implements acceptance tests

### PR DESCRIPTION
**Pull request by aminah-io**

### CODE REVIEW

File: api/scorer/test/test_create_community_id.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    # Implemented in previous step
+    pass
+
+
+@then("that Community is registered")
+def _(community_response):
+    """that Community is registered."""
+    assert community_response.status_code == 200
+    assert len(Community.objects.all()) == 1
+    # community = Community.objects.all()[0]
```
Comment (nutrina) [inline api/scorer/test/test_create_community_id.py:62]: Please also check the attributes (name & description) of the community instance

File: api/scorer/test/test_create_community_id.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+@scenario("features/create_community_id.feature", "Successfully create a Community ID")
+def test_successfully_create_a_community_id():
+    """Successfully create a Community ID."""
+
+
+@given("that I have an API account", target_fixture="account")
+def _(scorer_account, mocker):
+    """that I have an API account."""
+    mocker.patch("account.views.submit_signed_challenge", return_value=scorer_account)
```
Comment (nutrina) [inline api/scorer/test/test_create_community_id.py:27]: I don't think this mock is used

File: api/scorer/test/test_create_api_account.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    return account_response
+
+
+@then("I will have an account created", target_fixture="account_response")
+def _(account_response):
+    """I will have an account created."""
+
+    assert account_response.status_code == 200
+    assert len(Account.objects.all()) == 1
```
Comment (nutrina) [inline api/scorer/test/test_create_api_account.py:104]: Please check the account address, it should match the account that signed the message

File: api/scorer/test/test_create_api_account.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    """Successfully create an API account."""
+
+
+@given(
+    "that I am a builder and I don't have an API ACCOUNT",
+    target_fixture="no_account_db_response",
+)
+def _(no_account_db_response):
+    """that I am a builder and I don't have an API ACCOUNT."""
+    return no_account_db_response
```
Comment (nutrina) [inline api/scorer/test/test_create_api_account.py:36]: This is probably not needed, it does not seem to be used anywhere

File: api/scorer/test/test_create_api_keys.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    "features/create_api_keys.feature", "Successfully create API keys for my apps"
+)
+def test_successfully_create_api_keys_for_my_apps():
+    """Successfully create up to the maximum API keys for my apps."""
+
+
+@given("that I have an API account", target_fixture="account")
+def _(scorer_account, mocker):
+    """that I have an API account."""
+    mocker.patch("account.views.submit_signed_challenge", return_value=scorer_account)
```
Comment (nutrina) [inline api/scorer/test/test_create_api_keys.py:31]: Ido  not think we need this patch.

**Review by nutrina — CHANGES_REQUESTED** [review]
Just some minor observations
Please also add / update the cypress tests

**Review by schultztimothy — APPROVED** [review]
This looks great 🚀 . From what I'm seeing the changes requested by @nutrina have been resolved, everything is passing and the requested tickets are covered.

**Pull request merged**